native-script.rb

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+# Only the user can manually download and install Xcode from AppStore 
+puts "Installing Xcode. Please click on 'Get Xcode' on the following dialog and install Xcode from AppStore."
+`xcode-select --install`
+
+until `pkgutil --pkg-info=com.apple.pkg.Xcode`.include? "version" do
+  puts "Waiting for Xcode do finish installing..."
+  sleep(20)
+end
+
+puts "You need to agree to Xcode license to be able to use the compilers... (You might need to provide your password.)"
```

Comment:
You need to agree to the Xcode license agreement to be able to use the compilers. (You might need...)